wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandDecoratorTest.java

```diff
@@ -182,4 +183,26 @@ void perpetuallyTest() {
       assertTrue(scheduler.isScheduled(perpetual));
     }
   }
+
+  @Test
+  void unlessTest() {
+    try (CommandScheduler scheduler = new CommandScheduler()) {
+      AtomicBoolean unlessCondition = new AtomicBoolean();
+      AtomicBoolean hasRunCondition = new AtomicBoolean();
+      hasRunCondition.set(false);
+
+      Command command =
+          new InstantCommand(() -> hasRunCondition.set(true)).unless(unlessCondition::get);
+
+      unlessCondition.set(true);
```

Comment:
C++ initializes these as initial values at declaration; Java separately calls `set()`. It should be consistent.
If you missed it, `AtomicBoolean` should have a constructor accepting an initial value.